canvas_modules/harness/cypress/support/canvas/verification-cmds.js

```diff
@@ -1313,6 +1313,13 @@ Cypress.Commands.add("verifyNotificationCenterHidden", (hidden) => {
 	}
 });
 
+Cypress.Commands.add("verifyNodeWidthHeight", (nodeLabel, nodeWidth, nodeheight) => {
+	cy.getNodeWithLabel(nodeLabel).should((element) => {
+		expect(element).to.have.css("height", `${nodeheight}px`);
+		expect(element).to.have.css("width", `${nodeWidth}px`);
```

Comment:
Change these to add in the padding amounts passed in. By the way, nodeheight should have a capital H.